Infusionsoft/InvoiceServiceBase.php

```diff
@@ -272,11 +271,8 @@ public static function deleteInvoice($invoiceId, Infusionsoft_App $app = null){
             (int) $invoiceId
         );
 
-        $invoice = new Infusionsoft_Invoice($invoiceId);
-        $job = new Infusionsoft_Job($invoice->JobId);
         $result = parent::send($app, "InvoiceService.deleteInvoice", $params);
-        Infusionsoft_SdkEventManager::dispatch(new Infusionsoft_SdkEvent($invoice, array('result' => $result)), 'DataObject.Deleted');
-        Infusionsoft_SdkEventManager::dispatch(new Infusionsoft_SdkEvent($job, array('result' => $result)), 'DataObject.Deleted');
+        Infusionsoft_SdkEventManager::dispatch(new Infusionsoft_SdkEvent($result, array('result' => $result)), 'InvoiceService.OrderDeleted');

```

Comment:
$result is going to be a boolean here (I think). Better to send the $invoiceId that was passed in instead.